ops/deploy.sh

```diff
@@ -19,7 +19,29 @@ deploy_infra() {
 }
 
 deploy_e2e() {
-  echo "deploy_e2e"
+  sed -i -e "s/heartbeat_backend:latest/${AWS_ECR_HOST}\/heartbeat_backend:hb${BUILDKITE_BUILD_NUMBER}/g" ./ops/infra/docker-compose.yml
+  sed -i -e "s/heartbeat_frontend:latest/${AWS_ECR_HOST}\/heartbeat_frontend:hb${BUILDKITE_BUILD_NUMBER}/g" ./ops/infra/docker-compose.yml
+
+  scp -o StrictHostKeyChecking=no -i /var/lib/buildkite-agent/.ssh/HeartBeatKeyPair.pem -P "${AWS_SSH_PORT}" ./ops/infra/docker-compose.yml "${AWS_USERNAME}@${AWS_EC2_IP}:./"
+
+  echo "Start to deploy e2e service"
+  ssh -o StrictHostKeyChecking=no -i /var/lib/buildkite-agent/.ssh/HeartBeatKeyPair.pem -p "${AWS_SSH_PORT}" "${AWS_USERNAME}@${AWS_EC2_IP}" "
+    aws ecr get-login-password --region ${AWS_REGION} | docker login --username AWS --password-stdin ${AWS_ECR_HOST}
```

Comment:
pipeline has login aws when `source ./ops/base.sh` in this file, so sholuld not login again